CHANGELOG.md

```diff
@@ -1,6 +1,57 @@
 # CHANGELOG
 
-# 0.11.0 (2021-01-14)
+# [0.12.0 (YYYY-MM-DD) - `Set`, accessors, `@@toStringTag` and panic reduction](https://github.com/boa-dev/boa/compare/v0.11.0...v0.12.0)
```

Comment:
didn't panics increase :(

Reply:
Good point. So, many of the big panics were resolved, but some internal changes introduced some extra panics. Nevertheless, we still have a couple of changes in the pipeline that would reduce the panics.

Reply:
new plan: fix enough panics to make that statement true

Reply:
panic reduction? how about panic elimination!